src/helpers/geoChart.ts

```diff
@@ -69,3 +69,30 @@ export function getGeoDataIndex(buckets: VisualizationObject.IBucket[]): IGeoDat
 
     return geoDataIndex;
 }
+
+export function getGeoAttributeHeaderItems(
```

Comment:
Please consider this in my PR
https://github.com/gooddata/gooddata-react-components/pull/1315/files#diff-4e077217db022c533911725a328eb2feR158

Reply:
they are different. 1 used executionReponses, 1 used executionResults